### PR TITLE
[TASK] Add Danish dictionary compound word token filter #2974

### PR DESCRIPTION
**Pull request by leslawp**
# What this pr does

It adds vocabulary config and list that breaks down Danish compound words while indexing

# How to test

You can see example by making analyis: http://localhost:8983/solr/#/core_da/analysis?analysis.fieldvalue=projektudvikling&analysis.query=projekt%20udvikling&analysis.fieldtype=text&verbose_output=0
Look what DCWTF filter is doing (left side)

Fixes: #2974

### CODE REVIEW

**Review by dkd-kaehm — CHANGES_REQUESTED** [review]
Thanks for quick PR.
We want a little changes.

File: Resources/Private/Solr/configsets/ext_solr_11_1_0/conf/danish/schema.xml [hunk 1]
```diff
@@ -104,6 +111,13 @@
...
 
 			<filter class="solr.ManagedSynonymGraphFilterFactory" managed="${solr.core.name}" />
+			<filter class="solr.DictionaryCompoundWordTokenFilterFactory"
+					dictionary="danish/danish-common-nouns.txt"
+					minWordSize="5"
+					minSubwordSize="4"
+					maxSubwordSize="15"
+					onlyLongestMatch="false"
+			/>
 			<filter class="solr.FlattenGraphFilterFactory"/>
```
Comment (dkd-kaehm) [inline Resources/Private/Solr/configsets/ext_solr_11_1_0/conf/danish/schema.xml:121]: Same as above "DictionaryCompoundWordTokenFilterFactory after FlattenGraphFilterFactory"

Comment (leslawp) [reply Resources/Private/Solr/configsets/ext_solr_11_1_0/conf/danish/schema.xml]: Yes, so I just pushed (with force) fixed commit. I don't know if it is right thing as it is my first pull request. Maybe I should just add new commit with corrections??

**Review by dkd-kaehm — APPROVED** [review]
:+1: 
Thanks for fixing.

We'll test that in Analysis Solr Admin UI and merge it soonish.

**Comment by leslawp** [comment]
> Thanks for fixing.

Great then! Thank you!

**Review by dkd-kaehm — APPROVED** [review]
Tested with fortune combinations:

* krop krænkelse -> kropkrænkelse -> `kropkrank krop krank` 
* læse kredsløb -> læsekredsløb -> `lasekredslob las kredslob`
* låne lift -> lånelift -> `lanelift lan lift`
* udlån møbel -> udlånmøbel -> `udlanmobel udlan mobel`

:+1:

**Pull request merged**